crates/nu-plugin/src/protocol/mod.rs

```diff
@@ -29,10 +29,19 @@ pub enum PluginCall {
     CollapseCustomValue(PluginData),
 }
 
+/// An error message with debugging information that can be passed to Nushell from the plugin
+///
+/// The `LabeledError` struct is a structured error message that can be returned from
+/// a [Plugin's](crate::Plugin) [`run`](crate::Plugin::run()) method. It contains
```

Comment:
maybe
```diff
-/// a [Plugin's](crate::Plugin) [`run`](crate::Plugin::run()) method. It contains
+/// a [Plugin]'s [`run`](crate::Plugin::run()) method. It contains
```
? :yum: 
because right now the two links are blue, with the `'s` in the middle also blue, i find it a bit hard to see there are two links :open_mouth:

Reply:
thanks a lot :blush: